Sources/NIOSSH/SshAgent/AgentIdentity.swift

```diff
@@ -0,0 +1,108 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2019 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
```

Comment:
Prefer to import `FoundationEssentials` if it's available:

```suggestion
#if canImport(FoundationEssentials)
import FoundationEssentials
#else
import Foundation
#endif
```